content/en/docs/gke/customizing-gke.md

```diff
@@ -230,16 +266,6 @@ More detailed instructions follow.
       * one for GPU machines, in [`cluster.jinja`](https://github.com/kubeflow/manifests/tree/{{< params "githubbranch" >}}/gcp/deployment_manager_configs/cluster.jinja#L140).
```

Comment:
Line 228 (sorry, I couldn't select it)

- For consistency: `Google Cloud` ~~`Platform`~~/~~`(GCP)`~~, as you did in the other file.